### PR TITLE
Fix addEventListener call in PluggableMap

### PR DESCRIPTION
**Pull request by bartvde**
We found an issue in PluggableMap that can cause issues in a minified environment, this PR fixes the call to addEventListener to be explicit for window, which fixes the issue in our case

### CODE REVIEW

**Comment by ahocevar** [comment]
@bartvde To make the CI pass, please rebase this on top of master.

File: src/ol/PluggableMap.js [hunk 1]
```diff
@@ -1031,7 +1031,7 @@ class PluggableMap extends BaseObject {
 
       if (!this.handleResize_) {
         this.handleResize_ = this.updateSize.bind(this);
-        addEventListener(EventType.RESIZE, this.handleResize_, false);
+        window.addEventListener(EventType.RESIZE, this.handleResize_, false);
```
Comment (ahocevar) [inline src/ol/PluggableMap.js]: Would `global` instead of `window` also work?

Comment (bartvde) [reply src/ol/PluggableMap.js]: yes that should also do the trick, but we had other occurences of window in the source so I went with that

**Comment by bartvde** [comment]
I'll open up a clean PR off of master, didn't realize the same code is still in master

File: src/ol/PluggableMap.js (hunk 1 above)
Comment (bartvde) [reply src/ol/PluggableMap.js]: I verified that global also works, see #9601 for the update

**Pull request closed without merging**